.github/workflows/release-c7-data-migrator.yml

```diff
@@ -98,6 +98,15 @@ jobs:
         uses: ./.github/actions/setup-maven
         with:
           secrets: ${{ toJSON(secrets) }}
+          maven-servers: |
```

Comment:
Essential to fix https://github.com/camunda/c7-data-migrator/actions/runs/14972571655/job/42056745985